src/Sulu/Bundle/AdminBundle/Resources/js/components/Select/Select.js

```diff
@@ -0,0 +1,104 @@
+// @flow
+import React from 'react';
+import {action, computed, observable} from 'mobx';
+import {observer} from 'mobx-react';
+import selectStyles from './select.scss';
+import Divider from './Divider';
+import OverlayList from './OverlayList';
+import Option from './Option';
+import Label from './Label';
+
+@observer
+export default class Select extends React.PureComponent {
+    props: {
+        value?: string,
+        onChange?: (s: string) => void,
```

Comment:
😞 Well, then give it at least a better name, like `value`. And can you check if there is already an issue on https://github.com/reactjs/react-docgen/issues, resp. create one as suggested in the error, and link it here?